spark/src/main/java/org/apache/iceberg/actions/BaseSparkAction.java

```diff
@@ -86,6 +126,32 @@
     return manifestDF.union(otherMetadataFileDF).union(manifestListDF);
   }
 
+  protected static Dataset<Row> loadMetadataTable(SparkSession spark, String tableName, String tableLocation,
+                                                  MetadataTableType type) {
+    DataFrameReader noCatalogReader = spark.read().format("iceberg");
+    if (tableName.contains("/")) {
+      // Metadata location passed, load without a catalog
+      return noCatalogReader.load(tableName + "#" + type);
+    } else {
```

Comment:
nit: do we think this else helps clarity? We have a return statement anyway. Without this else, we could reduce the indentation of the block below.

Reply:
It helps for my brain,  but I can drop the if else if you like. I tend to prefer nesting over implied control flow breaks but I think I'm not in the majority here. I'll change it since its what we did on the other pr as well